cmd/cert-checker/main.go

```diff
@@ -139,6 +139,7 @@ func (c *certChecker) getCerts(unexpiredOnly bool) error {
 	var retries int
 	for {
 		sni, err = c.dbMap.SelectNullInt(
+			context.Background(),
```

Comment:
Honestly, this one and a few others here in cert-checker feel like exactly the right place for `context.TODO()`. We _should_ replace these with contexts that have actual timeouts eventually, based on a newly-introduced configurable cert-checker max-cycle-time or something like that.

Reply:
Modified cert-checker to plumb through context in more places, and to set context.TODO() at the top level.